### PR TITLE
[Javascript] Add nanoexpress

### PR DESCRIPTION
**Pull request by dalisoft**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [ ] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [x] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

File: javascript/nanoexpress/config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+framework:
+  github: dalisoft/nanoexpress
+  website: nanoexpress.js.org
```
Comment (waghanza) [inline javascript/nanoexpress/config.yaml]: could you use only one markup, rather `website`

File: javascript/nanoexpress/config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  version: 1.1.4
```
Comment (waghanza) [inline javascript/nanoexpress/config.yaml]: could you use X.Y format ?

File: javascript/nanoexpress/package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "^1.1.4"
```
Comment (waghanza) [inline javascript/nanoexpress/package.json]: could you use `~1.1.4` to keep in `1.1` ?

**Comment by dalisoft** [comment]
@waghanza Pro is licensed under GPL-3.0 and about code-changes

- Faster
- API more similar to Express than default version
- Less configurations than default version

All other things are same as default and code-base as well as plugins are backwards-compatible with default version.

File: javascript/nanoexpress/package.json (hunk 3 above)
Comment (dalisoft) [reply javascript/nanoexpress/package.json]: I fixed correctly? I want use always use latest ver in benchmarks

File: javascript/nanoexpress/package.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress": "^1.1"
```
Comment (waghanza) [inline javascript/nanoexpress/package.json]: rather use tilde @dalisoft 

to stick into the minor

use carret could allow usage of `1.2` or `1.9`

Comment (dalisoft) [reply javascript/nanoexpress/package.json]: I think @dependabot does the job for update, right?

I changed :)

Comment (waghanza) [reply javascript/nanoexpress/package.json]: Yes

**Comment by waghanza** [comment]
could you rebase and resolve conflict ?

either manually, or with `bin/make deps_config` :stuck_out_tongue:

**Comment by dalisoft** [comment]
just 5 min, i will do :)

**Comment by dalisoft** [comment]
done

**Pull request merged**